pkg/mqtrigger/messageQueue/nats/nats.go

```diff
@@ -57,7 +57,8 @@ func init() {
 }
 
 const (
-	natsProtocol          = "nats://"
+	// Commenting it to ensure verify-staticcheck passes
+	// natsProtocol          = "nats://"
```

Comment:
Instead of commenting, can we delete this line please?

Reply:
Done.